packages/synapse-react-client/src/components/SynapseTable/ExpandableTableDataCell.tsx

```diff
@@ -1,5 +1,5 @@
 import { useRef, useState } from 'react'
-import useResizeObserver from '@react-hook/resize-observer'
+import { useResizeObserver } from '@react-hookz/web'
```

Comment:
We are using `@react-hookz/web` for other utilities, so we might as well use this one too